src/smart-components/group/group.js

```diff
@@ -37,6 +41,14 @@ const Group = ({
   return (
     <Fragment>
       <TopToolbar breadcrumbs={ breadcrumbsList() }>
+        <Button className='edit-group-button' onClick={ () => setShowEdit(true) } variant='secondary'>Edit Group</Button>
+        <EditGroup
+          isOpen={ showEdit }
+          group={ group }
+          closeUrl={ `group/detail/${uuid}` }
+          onClose={ () => setShowEdit(false) }
+          postMethod={ () => new Promise(() => setShowEdit(false)) }
```

Comment:
No need to wrap in new Promise. Once promise is initiated it will always be thenable, this can be only `setShowEdit(false)`.

However I think that we should fetch current group once the edit is finished so we refresh the UI and let user know about these changes.